docs/xdr/features/collect/integrations/cloud_and_saas/salesforce.md

```diff
@@ -0,0 +1,33 @@
+uuid: 2b13307b-7439-4973-900a-2b58303cac90
+name: Salesforce
+type: intake
+
+## Overview
+
+Salesforce provides customer relationship management software and applications focused on sales, customer service, marketing automation, e-commerce, analytics, and application development.
+
+{!_shared_content/operations_center/detection/generated/suggested_rules_2b13307b-7439-4973-900a-2b58303cac90_do_not_edit_manually.md!}
+
+{!_shared_content/operations_center/integrations/generated/2b13307b-7439-4973-900a-2b58303cac90.md!}
+
+## Configure
+
+This setup guide will show you how to provide an integration between Salesforce events and SEKOIA.IO.
+
+### Prerequisites
+
+- Have an internal concentrator (Salesforce)
+
+### Configure OAuth
+
+1. Go to the [Salesforce App Manager](https://login.salesforce.com)
+2. Follow up [these instructions](https://help.salesforce.com/s/articleView?id=sf.connected_app_create_api_integration.htm&type=5) to configure new app
+
+   ***Note: After configuration is complete please check if your OAuth credentials are valid and you can interact with salesforce api***
+
+3. Get `clientId` and `clientSecret` from your app
+
+### Create an intake
+
+Go to the [intake page](https://app.sekoia.io/operations/intakes) and create a new intake from the format Salesforce.
+
```

Comment:
Please, don't forget to replace <name of the connector here> in the following suggestion. Thanks:
```suggestion
### Create an intake

Go to the [intake page](https://app.sekoia.io/operations/intakes) and create a new intake from the format Salesforce. Copy the intake key.

### Pull events

To start to pull events, you have to:

1. Go to the [playbooks page](https://app.sekoia.io/operations/playbooks) and create a new playbook with the [<name of the connector here>](../../../automate/library/salesforce.md) trigger
2. Set up the module configuration with the Github organization and the APIkey. Set up the trigger configuration with the intake key
3. Start the playbook and enjoy your events

```

Reply:
Thank you a lot! I've updated docs with your suggestions but will approve this only after testing